src/WindowsAzureMessaging/WindowsAzureMessaging/Models/MSNotificationHubMessage.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef MSNotificationHubMessage_h
+#define MSNotificationHubMessage_h
+
+#import <Foundation/Foundation.h>
+
+@interface MSNotificationHubMessage : NSObject
```

Comment:
We need to revisit this in subsequent previews. We need to ensure we are exposing _all_ of the underlying notification fields and we should avoid wrapping/abstracting the notification.